lms/static/js/learner_dashboard/views/progress_circle_view.js

```diff
@@ -0,0 +1,96 @@
+(function(define) {
```

Comment:
Thanks for making this a generic component! We'll be picking up a story soon to add this to the new course home page, so it will be great to be able to reuse it.

Since this is not a dashboard-specific feature, I'd move this to:

https://github.com/edx/edx-platform/tree/master/common/static/common/js/components/views

It can then load its templates from:

https://github.com/edx/edx-platform/tree/master/common/static/common/templates/components